docs/scss/_blog.scss

```
@@ -0,0 +1,33 @@
+.ov-masthead--home {
+    .ov-heading--lg {
+        max-width: 400px;
+        margin-left: auto;
+        margin-right: auto;
+    }
+}
+
+
+.ov-masthead ov-masthead--blog {
```

```suggestion
.ov-masthead--blog {
```
This class is considered a modifier so you don't need to add the `.ov-masthead` you can just style `.ov-masthead--blog` directly. 
If you want to learn more about how I wrote the classes checkout http://getbem.com/naming/